data/hooks/conf_regen/43-dnsmasq

```diff
@@ -50,6 +50,21 @@ do_pre_regen() {
 do_post_regen() {
   regen_conf_files=$1
 
+  # Fuck it, those domain/search entries from dhclient are usually annoying
+  # lying shit from the ISP trying to MiTM
+  if grep -q -E "^ *(domain|search)" /run/resolvconf/resolv.conf
```

Comment:
Whoukay... I didn't understand what I read :-/

Reply:
Meh yea I was probably a bit brutal here after being annoyed so many times with this ... The thing is that /etc/resolv.conf can contain `search` lines, for example `search foo`. Such that each time you query a domain, it will also test if `youdomain.foo` exists ... And typically this is set via the DHCPclient by shitty ISPs (for existence at home I have `search numericable`...) to lie about domains that are non-existent and I regularly end up with certificate warning when trying to reach a website because numericable is answering ... (And sometime even when contacting real existing domains because of idk temporary DNS resolution issue). It also happens on LXC/LXD which injects `search lxd` (not sure why, maybe to more easily contact other LXC...)

And it also messed up the check of the content of this file in the diagnosis ... We could also just ignore the corresponding lines but hmpf I just find this completely stupid and annoying so the following lines enforce that there's no `search` thing in the file)